spec/requests/stories_index_spec.rb

```diff
@@ -317,4 +317,14 @@ def create_live_sponsor(org, tag)
       end
     end
   end
+
+  describe "GET user_path" do
+    let(:user) { create(:user) }
+
+    it "redirects to the lowercase route for usernames" do
+      get "/#{user.username.capitalize}"
+      expect(response.body).to include("/#{user.username.downcase}")
+      expect(response.body).not_to include("/#{user.username.capitalize}")
```

Comment:
Note: I am unable to test the redirect here with `redirect_to` because the redirect happens in a `before_action`, which "halts the request cycle", as shown in the Rails docs: https://guides.rubyonrails.org/action_controller_overview.html#filters 😄

Reply:
Hmm, I was able to get it working. I also think it's curcial here we are testing the write http status.
```suggestion
      expect(response).to have_http_status(:moved_permanently)
      expect(response).to redirect_to("/#{user.username}")
```

Reply:
Hmm, so am I now... 🤔 Anyway, I think that's a great suggestion and I've made this change, but have also ensured that we're checking that the redirect goes to `/#{user.username.downcase}`. 😄